packages/app-cli/tests/services_SearchFilter.js

```diff
@@ -299,6 +299,72 @@ describe('services_SearchFilter', function() {
 		expect(ids(rows)).toContain(n3.id);
 	}));
```

Comment:
Could you see if you can convert this file to a ".ts"? No need to add types to everything - just the minimum so that it compiles. The main thing you'll need to do is convert the "require" to import. You do that by removing the .default and adding import/from. eg `const time = require('@joplin/lib/time').default;` => `import time from '@joplin/lib/time');`